### PR TITLE
Extract strum_macros::helpers into it's own module

### PR DESCRIPTION
**Pull request by ethankhall**
This was done so I could access the helpers in another project.

### CODE REVIEW

**Comment by ethankhall** [comment]
@Peternator7 any chance you'll be able to take a look at this sometime?

**Comment by Peternator7** [comment]
Hi @ethankhall, sorry for the delay here. The whole Covid thing has really messed up my schedule, and this got lost somewhere. Thanks for the ping :) 

Are you writing your own proc-macros? I'm not opposed to refactoring this code into it's own package, but I'd like to understand your use-case a little better to make sure this is really the best way of doing it.

Thanks!

**Comment by ethankhall** [comment]
No worries! Stuff falls through the cracks from time to time.

Sure, I can actually share what I'm doing with you. The goal was to make it easier to make CLI's with errors that generate error codes on failure.

Here's how I've been using my branch this change: https://github.com/ethankhall/error-enum-generator

File: strum_helpers/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+[lib]
+name = "strum_helpers"
+
+[dependencies]
+heck = "0.3"
+proc-macro2 = "1.0"
+syn = { version = "1.0", features = ["parsing", "extra-traits"] }
+
+[features]
+verbose-enumstring-name = []
```
Comment (Peternator7) [inline strum_helpers/Cargo.toml:24]: Do we need these feature flags on the helpers crate? I think they are just needed in the primary "macros" crate.

Comment (ethankhall) [reply strum_helpers/Cargo.toml:24]: Sorry life's been super busy. I don't think so, it was mostly copy-paste. I'll update the branch & pr here shortly.

**Pull request closed without merging**